asserts/snap_asserts.go

```diff
@@ -63,6 +64,11 @@ func (snapdcl *SnapDeclaration) Timestamp() time.Time {
 	return snapdcl.timestamp
 }
 
+// RefreshControl return the snaps whose updates are controlled by this declaration
```

Comment:
"*returns* the snaps ..." I think, and it would be nice to finish with a period / full stop.